deps/jerry/v8jerry/v8jerry_value.cpp

```diff
@@ -81,13 +81,14 @@ JerryValue* JerryValue::GetInternalProperty(JerryValue* key) {
 
 JerryValue* JerryValue::GetOwnPropertyDescriptor(const JerryValue& jkey) const {
     jerry_value_t key = jkey.value();
-    jerry_value_t descriptor = JerryIsolate::GetCurrent()->HelperGetOwnPropDesc().Call(m_value, &key, 1);
-
+    jerry_property_descriptor_t prop_desc;
+    jerry_get_own_property_descriptor (m_value, key, &prop_desc);
+    jerry_value_t descriptor = jerry_from_property_descriptor (&prop_desc);
     return new JerryValue(descriptor);
 }
 
 JerryValue* JerryValue::GetOwnPropertyNames() const {
-    return new JerryValue(JerryIsolate::GetCurrent()->HelperGetOwnPropNames().Call(m_value, NULL, 0));
+    return new JerryValue(jerry_object_get_property_names (m_value, JERRY_PROPERTY_FILTER_TRAVERSE_PROTOTYPE_CHAIN));
```

Comment:
What should happen with symbols?

Reply:
If this function called by `v8::Object::GetOwnPropertyNames` The prototype chain should not be traversed, symbols should be skipped and only enumerable properties should be listed. see (https://github.com/nodejs/node/blob/master/deps/v8/src/api/api.cc#L4446)

Reply:
i updated the pull request

Reply:
The flags are still not okay.